indy_node/test/nym_txn/test_create_did_without_endorser.py

```diff
@@ -101,27 +97,6 @@ def test_create_did_without_endorser_with_different_dest(looper, nym_txn_data, s
         sdk_get_and_check_replies(looper, [request_couple])
 
 
-def test_create_did_without_endorser_sig_count_2_one_on_ledger(looper, txnPoolNodeSet, nym_txn_data, sdk_pool_handle,
-                                                               sdk_wallet_trustee):
```

Comment:
Any reason for removal of this test? I think I saw similar test, but I'm not sure, and it was among unit tests, not integration. So, dropping comment here just in case.

Reply:
This test was just removed to another folder (as it requires endorsers now).